resources/default_contacts.json

```diff
@@ -10,7 +10,35 @@
         },
         "photo-path": "icon_wallet_avatar",
         "add-chat?": true,
-        "dapp?": true
+        "dapp?": true,
+        "has-global-command?": true,
+        "dapp-url":
+        {
+          "en": "https://status.im/dapps/wallet/"
+        },
+        "bot-url": "local://wallet-bot"
```

Comment:
We can actually change `local://` to `bot://` then it will make more sense, I suppose